src/btree/bt_curnext.c

```diff
@@ -606,10 +655,14 @@ __wt_btcur_next(WT_CURSOR_BTREE *cbt, bool truncating)
 		if (F_ISSET(cbt, WT_CBT_ITERATE_APPEND)) {
 			switch (page->type) {
 			case WT_PAGE_COL_FIX:
-				ret = __cursor_fix_append_next(cbt, newpage);
+				if ((ret = __cursor_fix_append_next(cbt,
+				    newpage)) == WT_PREPARE_CONFLICT)
```

Comment:
This line wrapping looks a bit wrong. Try moving the `cbt` to the second line as well.